docs/tutorials/how-to-sampler.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1d6b29b2",
+   "metadata": {},
+   "source": [
+    "# Get started with the Sampler primitive\n",
+    "\n",
+    "Learn how to set up and use the Sampler primitive program.\n",
+    "\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "The Sampler primitive lets you more accurately contextualize counts. It takes a user circuit as an input and generates an error-mitigated readout of quasiprobabilities. This enables you to more efficiently evaluate the possibility of multiple relevant data points in the context of destructive interference.  \n",
+    "\n",
+    "\n",
+    "## Prepare the environment\n",
+    "\n",
+    "1. Follow the steps in the [getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to get your Quantum Service instance ready to use.\n",
+    "2. You'll need at least one circuit to submit to the program. Our examples all have circuits in them, but if you want to submit your own circuit, you can use Qiskit to create one. To learn how to create circuits by using Qiskit, see the [Circuit basics tutorial](https://qiskit.org/documentation/tutorials/circuits/01_circuit_basics.html).\n",
+    "\n",
+    "\n",
+    "## Start a session\n",
+    "\n",
+    "With Qiskit Runtime primitives, we introduce the concept of a session or a factory that allows you to define a job as a collection of iterative calls to the quantum computer. When you start a session, it caches the data you send so it doesn't have to be transmitted to the Quantum Datacenter on each iteration.\n",
+    "\n",
+    "\n",
+    "### Specify program inputs\n",
+    "\n",
+    "The Sampler takes in:\n",
+    "* The **circuits** you want to investigate.\n",
+    "* The **parameters** input to evaluate the circuits.\n",
+    "* Optional: The **backend** to run on. If one is not specified, the least busy backend is used. To learn about choosing a backend, see [Choose a backend](/docs/quantum-computing?topic=quantum-computing-choose-backend).\n",
```

Comment:
```suggestion
    "* Optional: The **backend** to run on. If one is not specified, the least busy backend is used.\n",
```
Broken link.. we can remove since it's IBM Cloud specific.

Reply:
Same 👆